service/matching/matching_engine.go

```diff
@@ -1821,6 +1821,9 @@ pollLoop:
 
 		nexusReq := task.nexus.request.GetRequest()
 		nexusReq.Header[nexus.HeaderRequestTimeout] = time.Until(task.nexus.deadline).String()
+		if !task.nexus.operationDeadline.IsZero() {
+			nexusReq.Header[nexus.HeaderOperationTimeout] = time.Until(task.nexus.operationDeadline).String()
```

Comment:
Please format with https://github.com/nexus-rpc/sdk-go/blob/5f4d7fcf66ce9faf92ff883505e0e0951e145aee/nexus/api.go#L375 to conform with the spec.